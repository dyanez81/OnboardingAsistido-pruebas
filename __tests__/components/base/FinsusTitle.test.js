import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';
import FisnusTitle from 'components/base/FinsusTitle';

describe('Testing FinsusTitle component', () => {
  it('render component', () => {
    const wrapper = shallow(<FisnusTitle title={'Mi titulo'} />);

    expect(wrapper).toMatchSnapshot();
    wrapper.setProps({title: 'Otro titulo'});
    expect(wrapper).toMatchSnapshot();
  });
});
