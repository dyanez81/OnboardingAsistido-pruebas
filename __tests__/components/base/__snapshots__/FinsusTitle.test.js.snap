// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FinsusTitle component render component 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "white",
        "fontFamily": "Montserrat-Regular",
        "fontSize": 26,
        "textAlign": "center",
      },
      Object {},
    ]
  }
>
  Mi titulo
</Text>
`;

exports[`Testing FinsusTitle component render component 2`] = `
<Text
  style={
    Array [
      Object {
        "color": "white",
        "fontFamily": "Montserrat-Regular",
        "fontSize": 26,
        "textAlign": "center",
      },
      Object {},
    ]
  }
>
  Otro titulo
</Text>
`;
